[
   {
      "body" : "Similar case today:\r\n```\r\nERROR: AcceptToMemoryPoolWorker: CheckInputs: f62430326609f0ad79f9a7503ed59ede0ca6991513829337e4a210d22d1e618f, non-mandatory-script-verify-flag (unknown error) (code 64)\r\n```\r\n\r\nbc.info said High S... No other blockexplorer knows about the tx, so can't check myself.\r\n",
      "created_at" : "2016-02-10T12:59:34Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6862#issuecomment-182361249",
      "id" : 182361249,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6862",
      "updated_at" : "2016-02-10T12:59:34Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/182361249",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/6848764?v=3",
         "events_url" : "https://api.github.com/users/paveljanik/events{/privacy}",
         "followers_url" : "https://api.github.com/users/paveljanik/followers",
         "following_url" : "https://api.github.com/users/paveljanik/following{/other_user}",
         "gists_url" : "https://api.github.com/users/paveljanik/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/paveljanik",
         "id" : 6848764,
         "login" : "paveljanik",
         "organizations_url" : "https://api.github.com/users/paveljanik/orgs",
         "received_events_url" : "https://api.github.com/users/paveljanik/received_events",
         "repos_url" : "https://api.github.com/users/paveljanik/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/paveljanik/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/paveljanik/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/paveljanik"
      }
   },
   {
      "body" : "Was this not fixed in #7276?",
      "created_at" : "2016-02-10T13:13:08Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6862#issuecomment-182368068",
      "id" : 182368068,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6862",
      "updated_at" : "2016-02-10T13:13:15Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/182368068",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "Ah, @paveljanik's case is another bug.",
      "created_at" : "2016-02-10T13:16:28Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6862#issuecomment-182368965",
      "id" : 182368965,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6862",
      "updated_at" : "2016-02-10T13:16:28Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/182368965",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "Yes the issue seems to be different, \"No error\" versus \"unknown error\"",
      "created_at" : "2016-02-10T13:19:14Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6862#issuecomment-182369826",
      "id" : 182369826,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6862",
      "updated_at" : "2016-02-10T13:19:14Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/182369826",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
         "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
         "followers_url" : "https://api.github.com/users/laanwj/followers",
         "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
         "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/laanwj",
         "id" : 126646,
         "login" : "laanwj",
         "organizations_url" : "https://api.github.com/users/laanwj/orgs",
         "received_events_url" : "https://api.github.com/users/laanwj/received_events",
         "repos_url" : "https://api.github.com/users/laanwj/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/laanwj"
      }
   },
   {
      "body" : "I think the original issue here is fixed by #7276. I'm working on a fix for @paveljanik's bug.",
      "created_at" : "2016-02-10T13:20:25Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6862#issuecomment-182370162",
      "id" : 182370162,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6862",
      "updated_at" : "2016-02-10T13:20:25Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/182370162",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   }
]
